test/tools/yulInterpreter/EWasmBuiltinInterpreter.cpp

```diff
@@ -119,6 +119,8 @@ u256 EWasmBuiltinInterpreter::evalBuiltin(YulString _fun, vector<u256> const& _a
 		return arg[0] != arg[1] ? 1 : 0;
 	else if (_fun == "i64.eqz"_yulstring)
 		return arg[0] == 0 ? 1 : 0;
+	else if (_fun == "i64.clz"_yulstring)
+		return __builtin_clz(arg[0]);
```

Comment:
Please don't use some compiler-internal function.

Reply:
Should I implement clz for uint64 or you already know some existing one I can use?

Reply:
Either is fine, I guess.

Reply:
I can do it basing on https://foonathan.net/2016/02/implementation-challenge-2/
Up to you.

Reply:
I think just a loop over the 64 bits is fine.